README.md

```diff
@@ -23,9 +24,9 @@ The length of time to wait for completion. Values should contain a corresponding
 
 ### `scope`
 
-**Required**  The scope of the Okteto preview environment to create.
+The scope of the Okteto preview environment to create.
```

Comment:
Isn't it required anymore?

Reply:
AFAIK, it was never required. It defaults to `personal`

Reply:
you are right, it is required, but I think it should be optional. I will update this PR.
@ifbyol or do you remember a reason to make it required? in the CLI it is optional

Reply:
@pchico83  No, I don't remember any reason, I think defaulting to personal makes sense. Can you mentioning it on the field description?